contracts/StaderOracle.sol

```diff
@@ -119,6 +119,9 @@ contract StaderOracle is IStaderOracle, AccessControlUpgradeable, PausableUpgrad
         if (block.number < lastTrustedNodeCountChangeBlock + trustedNodeChangeCoolingPeriod) {
             revert CooldownNotComplete();
         }
+        if (trustedNodesCount == MIN_TRUSTED_NODES) {
```

Comment:
Maybe to be more on the safe side and to make this check even broader like
`if (trustesNodesCount <= MIN_TRUSTED_NODES)`

Reply:
`trustedNodesCount == MIN_TRUSTED_NODES` with this check we will never end up in a situation where trustedNodesCount < MIN_TRUSTED_NODES, and this is pre check before state gets update, so I think this check can serve our purpose.

Reply:
[11c68d0](https://github.com/stader-labs/ethx/pull/258/commits/11c68d04b5950000c5428b209a3cc78cc7371ab4)